contracts/sNOTE.sol

```diff
@@ -222,6 +206,26 @@ contract sNOTE is ERC20VotesUpgradeable, BoringOwnable, UUPSUpgradeable, Reentra
         _mint(msg.sender, bptAfter - bptBefore);
     }
 
+    function _exitPool(IAsset[] memory assets, uint256 bptExitAmount) internal {
+        BALANCER_VAULT.exitPool(
+            NOTE_ETH_POOL_ID,
+            address(this),
+            payable(msg.sender), // Owner will receive the underyling assets
+            IVault.ExitPoolRequest(
+                assets,
+                // Accept whatever NOTE/WETH we will receive here, since these
```

Comment:
We should allow the user to specify minETH, minNOTE and whether they want WETH or ETH. For the collateral withdraw, we can have the inputs be [0, 0] and ETH.

Reply:
Fixed and added mint/redeem events